deck/test/functional/cypress/integration/google/clone.spec.js

```diff
@@ -26,17 +29,19 @@ describe('google: Clone Modal Build custom instance type', () => {
     const imageDropdown = ReactSelect('v2-wizard-page[key=location] div.form-group:contains("Image")');
     imageDropdown.get();
     imageDropdown.toggleDropdown();
-    imageDropdown.type('ubuntu-1404-trusty-v20181002');
+    imageDropdown.type('ubuntu-1404-trusty-v20181002', { force: true });
```

Comment:
Is a force here needed?

Reply:
yes, unfortunately, it seems we need to force it, as React Select’s `<input>` element is not visible in the DOM